lib/components/molecules/Search/Search.style.js

```diff
@@ -0,0 +1,61 @@
+import { css } from 'styled-components';
+
+export default css`
+  ${props => `margin: ${props.theme.spacing.UNIT_2} ${props.theme.spacing.UNIT_2};`};
+
+  .search-box {
+    display: flex;
+    ${props => `
+    border-radius: ${props.theme.spacing.UNIT_3};
+    ${props.theme.border.FIELD_PRIMARY}
+    `};
+    width: 638px;
+    margin: 0 auto;
+  }
+
+  .search-txt {
+    ${props => `font-size: ${props.theme.spacing.UNIT_2};`};
+    border: none;
+    background: none;
+    outline: none;
+    flex-grow: 1;
+  }
+
+  .search-button {
+    display: flex;
+    overflow: hidden;
+  }
+
+  button {
```

Comment:
This is wrong don't style the component here

Reply:
Once Icon componet will be exposed then unnecessary codes will be removed.